content/blog/ultimate-windows-development-environment-guide/index.md

```diff
@@ -0,0 +1,612 @@
+
+![What it looks like with more than one desktop](./multiple_desktops.png)
+
+In order to move a window from one desktop to another, simply drag a window into another desktop's preview window.
+
+![What it looks like when you drag a window to a new desktop](./dragging_windows_virtual_desktop.png)
+
+Finally, to delete a virtual desktop, you can hover over the preview of the desktop. Upon doing so, a close button should appear. Click it to close the virtual desktop instance.
+
+![A close button shows up when hovering over a virtual desktop](./close_virtual_desktop.png)
+
+> A feature that's soon-to-release is renaming a virtual desktop! This functionality is [being added in the 2020 stable release of Windows](https://blogs.windows.com/windowsexperience/2019/09/06/announcing-windows-10-insider-preview-build-18975/) launching soon!
+
+### Touchpad Users {#virtual-desktop-touchpad-users}
+
+If you're a laptop user (or have a touchpad for your desktop) that supports Windows gestures, you can configure a three or four finger feature to switch desktops with a simple swipe. Simply go to "Settings > Devices > Touchpad" to see if you're able to configure this or not. If you are able to, you should be able to select dropdowns to configure which one you'd like.
+
+![A preview of the touchpad settings page](./touchpad_virtual_desktop.png)
+
+
+
+## Symbolic Links {#symlinks}
+
+Symbolic links are a method of having a shortcut of sorts from one file/folder to another. Think of it as Windows Shortcuts but baked directly into the filesystem level. This may come as a surprise to some developers, but Windows actually has support for symbolic links!
+
+To use symbolic links from the CLI, you have to first enable developer mode on your install of Windows. To do this, go to your settings app, open "Update & Security," then select in the sidebar "For developers."
+
+![The settings page for "developer mode"](./developer_mode.png)
+
+Once done, you're able to run `mklink`, which provides you the ability to make a symbolic link.
+
+### Usage {#using-mklink}
+
+By default, it creates a soft link from the first argument to the second. 
+```
+mklink Symlink SourceFile
+```
+
+You're also able to add `/D` to make a soft link to a directory:
+```
+mklink /D SymlinkDir SourceFolder
+```
+
+Finally, to make hard links, you use `/H` for files:
+
+```
+mklink /H Symlink SourceFile
+```
+
+And `/J` for folders:
+
+```
+mklink /J SymlinkDir SourceFolder
+```
+
+### GUI Alternative {#link-shell-extension}
+
+While the CLI enables you to make hard and soft symbolic links, it's far from graceful. It'd be ideal to have that functionality baked right into the explorer menu options if used frequently. Luckily for us, there's an app for that! [Link Shell Extension](https://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html) adds the options to the context menu itself. It's even able to be installed using [Chocolatey](#package-management):
```

Comment:
```suggestion
While the CLI enables you to make hard and soft symbolic links, it's far from graceful. It would be ideal to have that functionality baked right into the explorer menu options if used frequently. Luckily for us, there's an app for that! [Link Shell Extension](https://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html) adds the options to the context menu itself. It's even able to be installed using [Chocolatey](#package-management):
```